tests/scripts/deploy-validate-vault.sh

```diff
@@ -189,17 +189,24 @@ function set_up_vault_kubernetes_auth {
   kubectl exec -ti vault-0 -- vault auth enable kubernetes
 
   # To fetch the service account issuer
-  kubectl proxy &
+  minikube kubectl proxy &
```

Comment:
```suggestion
  kubectl proxy &
```
why we need minikube as prefix in this command only?

Reply:
So we match with the server version, it's cleaner.